src/fparser/one/tests/test_do.py

```diff
@@ -0,0 +1,212 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please name this file with the rule name and number (see item 6. of the review guidelines).

Reply:
Item 6 seems to refer to the Mk.II parser and Fortran 2003 syntax. But since you asked, I have followed the same convention for this one.

Reply:
Great, thanks.